lisp/c/specials.c

```diff
@@ -555,6 +555,24 @@ register pointer arg;
   throw(ctx,tag,result);
   error(E_NOCATCHER,tag);}
 
+pointer MACROLET(ctx,arg)
+register context *ctx;
+register pointer arg;
+{ register pointer fns, fn;
```

Comment:
Unless I'm wrong, that `fn` stand for a macro.. did you consider giving it a name such as "mac", or even, spelled out, "macro"? reading "fn" makes me think of a function

Reply:
yes, you are right. It is set to `fn` because it was just copied from `flet`, but it should be adapted to `mac`-like names.

Will change this in a moment.